src/hotspot/share/gc/shenandoah/shenandoahReferenceProcessor.cpp

```diff
@@ -316,7 +316,7 @@ void ShenandoahReferenceProcessor::make_inactive(oop reference, ReferenceType ty
     reference_set_next(reference, reference);
   } else {
     // Clear referent
-    reference_set_referent(reference, NULL);
+    reference_clear_referent(reference);
```

Comment:
Now I am looking at this code and wonder if we could just inline `reference_clear_referent` and `reference_set_next` both in Shenandoah and ZGC code. Probably something for a followup.

Reply:
I'd prefer to keep the `reference_*` helper functions as is in ZGC.